egs/swbd/s5c/local/reverberate/prep_sim_rirs.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# Copyright 2016  Tom Ko
+# Apache 2.0
+# script to simulate room impulse responses (RIRs)
+
+sampling_rate=8000
+output_bit=16
+num_room=30
+rir_per_room=2
+
+. cmd.sh
+. path.sh
+. ./utils/parse_options.sh
+
+if [ $# != 1 ]; then
+  echo "Usage: "
+  echo "  $0 [options] <output-dir>"
+  echo "e.g.:"
+  echo " $0  data/impulses_noises"
+  exit 1;
+fi
+
+output_dir=$1
+mkdir -p $output_dir
+
+cat >./genrir.m <<EOF
```

Comment:
Not a good idea to have such huge inline scripts. There is a lot of scope for bugs. Save this file separately.